<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnswersWindow</class>
 <widget class="QWidget" name="AnswersWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AnswersWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Search:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="searchEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="questionEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
